apps/bc/contracts/BC.sol

```diff
@@ -0,0 +1,353 @@
+/*
+ * SPDX-License-Identitifer:    GPL-3.0-or-later
+ */
+
+pragma solidity ^0.4.24;
+
+import "@aragon/os/contracts/apps/AragonApp.sol";
+import "@aragon/os/contracts/common/EtherTokenConstant.sol";
+import "@aragon/os/contracts/common/IsContract.sol";
+import "@aragon/os/contracts/common/SafeERC20.sol";
+import "@aragon/os/contracts/lib/token/ERC20.sol";
+import "@aragon/os/contracts/lib/math/SafeMath.sol";
+
+import "@aragon/apps-token-manager/contracts/TokenManager.sol";
+import "@aragon/apps-vault/contracts/Vault.sol";
+import "@aragonblack/fundraising-pool/contracts/Pool.sol";
+
+import "./BancorContracts/converter/BancorFormula.sol";
+
+import "./IBC.sol";
+
+contract BC is EtherTokenConstant, IsContract, AragonApp, IBC, BancorFormula {
```

Comment:
i think the bonding curve should also implement the `TokenManager` interface, right?